lib/features/email/presentation/controller/email_controller.dart

```diff
@@ -137,22 +146,34 @@ class EmailController extends BaseController with AppLoaderMixin {
   @override
   void onClose() {
     _downloadProgressStateController.close();
+    _getEmailDeBouncer.cancel();
     _clearWorker();
     super.onClose();
   }
 
+  void _initializeDebounceTimeIndexPageViewChange() {
+    _getEmailDeBouncer.values.listen((value) async {
+      if(value !=null) {
+        _getEmailContentAction(emailParentController.listEmail[value].id);
+      }
+    });
+  }
+
   void _initWorker() {
-    emailWorker = ever(mailboxDashBoardController.selectedEmail, (presentationEmail) {
+    emailWorker = ever(emailParentController.selectedEmail, (presentationEmail) {
       log('EmailController::_initWorker(): $presentationEmail');
       if (presentationEmail is PresentationEmail) {
         if (_currentEmailId != presentationEmail.id) {
           _currentEmailId = presentationEmail.id;
+          emailParentController.setCurrentPositionEmailInListEmail(_currentEmailId);
```

Comment:
why not set it in `parent` controller?

Reply:
because action will call when apply current single email